gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
+    layouts = {
```

Comment:
...and what should it do if the 7-seg is displaying a combination that doesn't map to a character?